inbm-vision/node-agent/node/constant.py

```diff
@@ -99,3 +99,6 @@
 CONFIG_HEARTBEAT_RESPONSE_SECS_UPPER_LIMIT = 1800
 CONFIG_HEARTBEAT_RESPONSE_SECS_LOWER_LIMIT = 90
 CONFIG_HEARTBEAT_RESPONSE_SECS_DEFAULT = 300
+
+# Var directory
+VAR_DIR = '/var'
```

Comment:
Is there any chance this could run on Windows? If so, this should be a Windows path on Windows systems.

Reply:
Hi @gblewis1, node agent only run on Yocto image. I didn't heard any plan to run node agent in Windows system.